gobblin-core/src/main/java/gobblin/publisher/TimestampDataPublisher.java

```diff
@@ -13,15 +13,17 @@
 package gobblin.publisher;
 
 import java.io.IOException;
-
+import java.util.Set;
 import org.apache.hadoop.fs.Path;
-
+import com.google.common.base.Preconditions;
+import gobblin.util.ParallelRunner;
+import gobblin.util.WriterUtils;
 import gobblin.configuration.State;
 import gobblin.configuration.WorkUnitState;
 
 
 /**
- * Write output to {finaldir}/{currentTimestamp}
+ * Write output to {finaldir}/{dbname.tablename}/{currenttimestamp}
```

Comment:
Please have a more detailed description of:
- Preconditions: what is the writer output expected to be
- Post conditions: what does published output look like